jiant/tasks/lm.py

```diff
@@ -246,16 +246,25 @@ def update_metrics(self, out, batch=None):
         return
 
     def get_data_iter(self, path):
-        """Loading data file and tokenizing the text
+        """
+        Loading data file and tokenizing the text. We treat the Wikipedia corpus as a
+        long sequence, and we take each slice of 510 tokens as an example. The dataset
```

Comment:
Is it always 510, or does it depend on the specified max seq length?

Reply:
Ah.